exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
+	// e.g. 192.168.0.2:8080
+	EndpointHostPort string
+
+	// ReporterURI is the HTTP endpoint to which spans will be sent
+	// e.g. http://localhost:9411/api/v2/spans
+	ReporterURI string
+
+	// ReporterOptions are optional Options
+	// that modify the zipkin http Reporter.
+	ReporterOptions []ziphttp.ReporterOption
+}
+
+func NewExporterWithOptions(o *Options) (*Exporter, error) {
+	if o == nil {
+		// TODO: Perhaps fill in the defaults that OpenZipkin uses
+		o = new(Options)
+	}
+	if _, err := url.Parse(o.ReporterURI); err != nil {
+		return nil, fmt.Errorf("Failed to parse ReporterURI err: %v", err)
+	}
+	reporter := ziphttp.NewReporter(o.ReporterURI, o.ReporterOptions...)
+	localEndpoint, err := openzipkin.NewEndpoint("server", o.EndpointHostPort)
```

Comment:
why is "server" a good default here?

Reply:
I don't know, that's the default that's used in NewExporter. /cc @rakyll @jcd2 @adriancole

Reply:
I think the best is to use the name of the program by default, check `os.Args`.

Reply:
os.Args[0] sounds better.